test/run_test.py

```diff
@@ -1180,18 +1181,21 @@ def parse_args():
         and (
             TEST_WITH_CROSSREF
             or TEST_WITH_ASAN
-            or (
-                strtobool(os.environ.get("TD_DISTRIBUTED", "False"))
-                and TEST_CONFIG == "distributed"
-                and TEST_CUDA
-            )
+            or (TEST_CONFIG == "distributed" and TEST_CUDA)
             or (IS_WINDOWS and not TEST_CUDA)
             or TEST_CONFIG == "nogpu_AVX512"
             or TEST_CONFIG == "nogpu_NO_AVX2"
+            or (
+                "sm86" not in BUILD_ENVIRONMENT
```

Comment:
Ok, the logic to check for "sm86" here looks a bit awkward I think because that can be changed in the future.  But this is probably ok because I assume this is only temporary

Reply:
yeah all of the logic here is a bit cursed